modules/animation/src/animationcontroller.cpp

```diff
@@ -369,9 +378,27 @@ void AnimationController::render() {
         }
         app_->processEvents();
 
-        for (auto&& [exporter, base] : exporters) {
+        for (auto&& [exporterKinds, base] : exporters) {
             const auto file = fmt::format("{}{:0{}}", base, currentFrame, digits);
-            exporter->exportFile(renderLocation.get(), file, {writer.get()}, Overwrite::Yes);
+            std::visit(util::overloaded{
+                           [&](Exporter* exporter) {
+                               exporter->exportFile(renderLocation.get(), file, {writer.get()},
+                                                    Overwrite::Yes);
+                           },
+                           [&](CanvasProcessor* cp) {
+                               // Hackish: Make sure LayerRAM is the last valid rep, so that
+                               // is the one that will be cloned, This also forces the
+                               // download to happen on the main thread instead of in the back
+                               // ground.
```

Comment:
```suggestion
                               // Hackish: Make sure LayerRAM is the last valid rep, so that it
                               // is the one that will be cloned. This also forces the
                               // download to happen on the main thread instead of in the 
                               // background.
```